### PR TITLE
[Build-trigger-45] Providing more robust messages

### PR DESCRIPTION
**Pull request by khermano**
resolves https://github.com/jboss-set/build-trigger/issues/45

### CODE REVIEW

File: src/main/java/org/jboss/set/BuildTriggerResource.java [hunk 1]
```diff
@@ -38,27 +38,35 @@ public class BuildTriggerResource {
...
-            return Response.status(400, "List of streams is empty").build();
+            return Response.status(400, "The list of streams cannot be null or empty. "
+                    + "Please provide valid streams.").build();
         }
         logger.infof("Triggering build for product %s by %s", buildInfo, email);
 
         buildTrigger.triggerBuild(BuildJMSTriggerPayload.from(buildInfo, email));
 
-        return Response.ok("Triggered build for " + buildInfo.gitRepo + ":" + buildInfo.projectVersion).build();
+        return Response.ok("Build triggered successfully for repository " + buildInfo.gitRepo
```
Comment (petrberan) [inline src/main/java/org/jboss/set/BuildTriggerResource.java:48]: Does it even make sense to have these string messages here? I'm not sure if we even use them on FE.

@xjusko

Comment (xjusko) [reply src/main/java/org/jboss/set/BuildTriggerResource.java:48]: Yes, the response text is currently being used in notification alerts.

**Comment by xjusko** [comment]
LGTM, thanks.

**Review by petrberan — APPROVED** [review]
LGTM

**Pull request merged**